test/integration/utils/transactions.js

```diff
@@ -15,6 +15,7 @@
 'use strict';
 
 var lisk = require('lisk-elements').default;
+var getTransaction = require('./http').getTransaction;
 
 module.exports = {
 	generateValidTransaction() {
```

Comment:
this function is not being used in integration test anywhere, you can remove this.